releng/sap-all-ephemeral/pom.xml

```diff
@@ -84,14 +84,35 @@
 		 	<groupId>org.eclipse.dirigible</groupId>
```

Comment:
Could you apply the same changes also for the:
- releng/sap-all
- releng/sap-cms

All `Neo` related modules are:
- releng/sap-all
- releng/sap-all-ephemeral
- releng/sap-cms